src/views/Actions.vue

```diff
@@ -6,18 +6,57 @@
       <a href="#/results">Results</a>
     </div>
   </div>
-      <div class="results-message-wrapper">
-        <p>Earth is about the daily actions that enables responsible living.</p>
-        <p>Based on your responses, we've complied a list of actions you can do in Singapore...</p>
+    <div class="first-page-content" v-if="firstPage">
+      <p>Earth is about the daily actions that enables responsible living.</p>        
+      <p>Based on your responses, we've complied a list of actions you can do in Singapore...</p>
+      <div class="next" @click="changePage"><v-icon class="next-content" name="arrow-right"/></div>
+    </div>
+    <div class="second-page-content" v-else>
```

Comment:
can consider calling this `actions`/`actions-pledge`

later when this grows to more cards, move it into its own component